### PR TITLE
DSPDC-976 Generate a new Helm index as part of releasing charts.

### PR DESCRIPTION
**Pull request by danxmoran**
~We'll need to do some copy-paste for the code that commits the generated `index.yaml` to the `gh-pages` branch, because I don't think that works very well from within `sbt`.~

This now does all steps required to update the index, including git-fu.

### CODE REVIEW

**Comment by danxmoran** [comment]
Converting to draft while I look into exactly how hard it'd be to do the last git steps from sbt

**Comment by danxmoran** [comment]
Turns out the git-fu in sbt actually works!

File: plugins/helm/src/main/scala/org/broadinstitute/monster/sbt/MonsterHelmPlugin.scala [hunk 1]
```diff
@@ -68,29 +68,78 @@ object MonsterHelmPlugin extends AutoPlugin {
...
+          "-o" -> org,
+          "-r" -> repo,
+          "-p" -> packageDir,
+          "-i" -> indexTarget.getAbsolutePath()
+        )
+      )
+
+      val gitBase = (ThisBuild / baseDirectory).value
+
+      def git(cmd: String, args: String*): Unit = {
```
Comment (raaidbroad) [inline plugins/helm/src/main/scala/org/broadinstitute/monster/sbt/MonsterHelmPlugin.scala:107]: any point in having a more generic function that `cr` and `git` inherit from/are based off of? Structurally similar and both have the exit code check, but I understand that this is pretty tightly scoped rn so there might not be too much value in doing that. Just a nitpick 🙂

Comment (danxmoran) [reply plugins/helm/src/main/scala/org/broadinstitute/monster/sbt/MonsterHelmPlugin.scala:107]: I think there's enough difference that trying to consolidate the two wouldn't really be worth it.

**Pull request merged**